modules/gltf/editor_scene_exporter_gltf_plugin.h

```diff
@@ -31,11 +31,11 @@
 #ifndef EDITOR_SCENE_EXPORTER_GLTF_PLUGIN_H
 #define EDITOR_SCENE_EXPORTER_GLTF_PLUGIN_H
 
+#include "editor_scene_importer_gltf.h"
```

Comment:
That seems incorrect? Editor code only makes sense to include when `TOOLS_ENABLED` is defined.

Reply:
Isn't this entire file (`editor_scene_exporter_gltf_plugin.h`) already only for editor situations, so it wouldn't matter if `editor_scene_importer_gltf.h` is always included in it?

Reply:
In theory yes, in practice no, the `register_types.cpp` for the gltf module doesn't properly put editor-specific includes within `TOOLS_ENABLED`. And since all `.cpp` files get globbed in the `SCsub`, this _will_ still be compiled so it also needs `#ifdef TOOLS_ENABLED` guards.

Reply:
Ok, moved it under the `#ifdef TOOLS_ENABLED` line.